### PR TITLE
EG-3905 Fixed typo in checkpoint tooling pages for CE 4.1 - 4.5

### PR DESCRIPTION
**Pull request by ivanterziev-r3**
EG-3905 Fixed typo in checkpoint tooling pages for CE 4.1 - 4.5

### CODE REVIEW

File: content/en/docs/corda-enterprise/4.5/checkpoint-tooling.md [hunk 1]
```diff
@@ -60,12 +60,12 @@ To retrieve this information, execute `checkpoints dump` in the node’s shell.
 Below are some of the more important fields included in the output:
 
 
-* `flowId`: The id of the flow
-* `topLevelFlowClass`: The name of the original flow that was invoked (by RPC or a service)
-* `topLevelFlowLogic`: Detailed view of the top level flow
-* `flowCallStackSummary`: A summarised list of the current stack of sub flows along with any progress tracker information
-* `suspendedOn`: The command that the flow is suspended on (e.g. `SuspendAndReceive`) which includes the `suspendedTimestamp`
-* `flowCallStack` A detailed view of the of the current stack of sub flows
+* `flowId`: The id of the flow,
```
Comment (taylorharvey-r3) [inline content/en/docs/corda-enterprise/4.5/checkpoint-tooling.md:63]:
```suggestion
* `flowId`: The id of the flow.
```

File: content/en/docs/corda-enterprise/4.5/checkpoint-tooling.md [hunk 2]
```diff
@@ -60,12 +60,12 @@ To retrieve this information, execute `checkpoints dump` in the node’s shell.
...
+* `topLevelFlowClass`: The name of the original flow that was invoked (by RPC or a service),
```
Comment (taylorharvey-r3) [inline content/en/docs/corda-enterprise/4.5/checkpoint-tooling.md:64]:
```suggestion
* `topLevelFlowClass`: The name of the original flow that was invoked (by RPC or a service).
```

File: content/en/docs/corda-enterprise/4.5/checkpoint-tooling.md [hunk 3]
```diff
@@ -60,12 +60,12 @@ To retrieve this information, execute `checkpoints dump` in the node’s shell.
...
+* `topLevelFlowLogic`: Detailed view of the top level flow,
```
Comment (taylorharvey-r3) [inline content/en/docs/corda-enterprise/4.5/checkpoint-tooling.md:65]:
```suggestion
* `topLevelFlowLogic`: Detailed view of the top level flow.
```

File: content/en/docs/corda-enterprise/4.5/checkpoint-tooling.md [hunk 4]
```diff
@@ -60,12 +60,12 @@ To retrieve this information, execute `checkpoints dump` in the node’s shell.
...
+* `flowCallStackSummary`: A summarised list of the current stack of sub flows along with any progress tracker information,
```
Comment (taylorharvey-r3) [inline content/en/docs/corda-enterprise/4.5/checkpoint-tooling.md:66]:
```suggestion
* `flowCallStackSummary`: A summarised list of the current stack of sub flows along with any progress tracker information.
```

File: content/en/docs/corda-enterprise/4.5/checkpoint-tooling.md [hunk 5]
```diff
@@ -60,12 +60,12 @@ To retrieve this information, execute `checkpoints dump` in the node’s shell.
...
+* `suspendedOn`: The command that the flow is suspended on (for example, `SendAndReceive`), which includes the `suspendedTimestamp`,
```
Comment (taylorharvey-r3) [inline content/en/docs/corda-enterprise/4.5/checkpoint-tooling.md:67]:
```suggestion
* `suspendedOn`: The command that the flow is suspended on (for example, `SendAndReceive`), which includes the `suspendedTimestamp`.
```

File: content/en/docs/corda-enterprise/4.5/checkpoint-tooling.md [hunk 6]
```diff
@@ -60,12 +60,12 @@ To retrieve this information, execute `checkpoints dump` in the node’s shell.
...
+* `flowCallStack` A detailed view of the of the current stack of sub flows,
```
Comment (taylorharvey-r3) [inline content/en/docs/corda-enterprise/4.5/checkpoint-tooling.md:68]:
```suggestion
* `flowCallStack` A detailed view of the of the current stack of sub flows.
```

**Review by AMcFarlandCampbellR3 — APPROVED** [review]

**Review by taylorharvey-r3 — APPROVED** [review]
Suggested a few changes from commas to periods. Other than that LGTM.

**Review by taylorharvey-r3 — APPROVED** [review]
LGTM

**Pull request merged**